_docs/style/text-conventions/date-time.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Date and Time
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Rackspace is a global company, so it is critical that we represent date and time as clearly and consistently as possible to our customers.
+resource: false
+status: complete
+last-modified: 2018-06-28
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Date
+
+Dates are displayed differently in different countries, so you must use a date format that's explicit and consistent and that global users can't misinterpret.
+
+Unless space is limited, always show dates in the following format: month day, year. Always spell out the month.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| November 12, 2018 | November 12th, 2018<br />11/12/2018<br />11/12/18<br />11-12-18<br />12/11/18 |
```

Comment:
Design question: Considering we have users around the world, would it be a good idea to allow 12 November 2018 or 2018 November 12 as well for international usability? https://en.wikipedia.org/wiki/Date_format_by_country

Reply:
Adding @palindromebart

Reply:
@nimbinatus, I spoke with @palindromebart and he mentioned that he did reach out to dev about this question and the response back is that we do not localize dates in the UI.